Pico build fails

I'm trying out the Pico support on macOS. The setup succeeds without an error: `xs-dev setup --device=pico`. Running helloworld compiles the C binaries, builds the XS byte code, and links. All good.

Then it fails here:

```console
make: /Users/hoddie/pico/pico-sdk/build/elf2uf2/elf2uf2: No such file or directory
make: *** [/Users/hoddie/.local/share/moddable/build/bin/pico/debug/helloworld/xs_pico.uf2] Error 1
    at makeError (/usr/local/lib/node_modules/xs-dev/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/usr/local/lib/node_modules/xs-dev/node_modules/execa/index.js:118:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
  shortMessage: 'Command failed with exit code 2: mcconfig -d -m -p pico',
```

There is an older copy of the `pico-sdk` on my computer at that path, but it is not linked to by any environment variable. The copy of `pico-sdk` installed by xs-dev is at `/Users/hoddie/.local/share/pico/pico-sdk` and does appear to contain `elf2uf2` at the expected path.

--- a/src/commands/teardown.ts
+++ b/src/commands/teardown.ts
@@ -19,6 +19,8 @@ const command: GluegunCommand = {
     filesystem.remove(filesystem.resolve(INSTALL_DIR, 'wasm'))
     filesystem.remove(filesystem.resolve(INSTALL_DIR, 'esp32'))
     filesystem.remove(filesystem.resolve(INSTALL_DIR, 'esp'))
+    filesystem.remove(filesystem.resolve(INSTALL_DIR, 'pico'))
+    filesystem.remove(filesystem.resolve(INSTALL_DIR, 'fontbm'))
 
     await patching.patch(PROFILE_PATH, {
       delete: `source ${EXPORTS_FILE_PATH}`,

--- a/src/toolbox/setup/pico.ts
+++ b/src/toolbox/setup/pico.ts
@@ -10,9 +10,9 @@ export default async function (): Promise<void> {
   const PICO_SDK_REPO = 'https://github.com/raspberrypi/pico-sdk'
   const PICO_EXAMPLES_REPO = 'https://github.com/raspberrypi/pico-examples'
   const PICO_DIR = filesystem.resolve(INSTALL_DIR, 'pico')
-  const PICO_SDK_PATH = filesystem.resolve(PICO_DIR, 'pico-sdk')
+  const PICO_SDK_DIR = filesystem.resolve(PICO_DIR, 'pico-sdk')
   const PICO_EXAMPLES_PATH = filesystem.resolve(PICO_DIR, 'pico-examples')
-  const PICO_SDK_BUILD_DIR = filesystem.resolve(PICO_SDK_PATH, 'build')
+  const PICO_SDK_BUILD_DIR = filesystem.resolve(PICO_SDK_DIR, 'build')
 
   const spinner = print.spin()
   spinner.start('Starting pico tooling setup')
@@ -51,13 +51,13 @@ export default async function (): Promise<void> {
   spinner.succeed()
 
   // 2. Install the pico sdk and examples:
-  if (filesystem.exists(PICO_SDK_PATH) === false) {
+  if (filesystem.exists(PICO_SDK_DIR) === false) {
     spinner.start('Cloning pico-sdk repo')
-    await system.exec(`git clone -b master ${PICO_SDK_REPO} ${PICO_SDK_PATH}`, {
+    await system.exec(`git clone -b master ${PICO_SDK_REPO} ${PICO_SDK_DIR}`, {
       stdout: process.stdout,
     })
     await system.exec(`git submodule update --init`, {
-      cwd: PICO_SDK_PATH,
+      cwd: PICO_SDK_DIR,
       stdout: process.stdout,
     })
     spinner.succeed()
@@ -73,12 +73,12 @@ export default async function (): Promise<void> {
   }
 
   // 3. Set the PICO_SDK_PATH environment variable to point to the Pico SDK directory
-  if (process.env.PICO_SDK_PATH === undefined) {
-    spinner.info('Setting PICO_SDK_PATH')
-    process.env.PICO_SDK_PATH = PICO_SDK_PATH
-    await upsert(EXPORTS_FILE_PATH, `export PICO_SDK_PATH=${PICO_SDK_PATH}`)
+  if (process.env.PICO_SDK_DIR === undefined) {
+    spinner.info('Setting PICO_SDK_DIR')
+    process.env.PICO_SDK_DIR = PICO_SDK_DIR
+    await upsert(EXPORTS_FILE_PATH, `export PICO_SDK_DIR=${PICO_SDK_DIR}`)
   } else {
-    spinner.info(`Using existing $PICO_SDK_PATH: ${process.env.PICO_SDK_PATH}`)
+    spinner.info(`Using existing $PICO_SDK_DIR: ${process.env.PICO_SDK_DIR}`)
   }
 
   // 4. Build some pico tools:

